### PR TITLE
Project reincarnation

### PR DESCRIPTION
**Pull request by MaticDiba**
Maybe ... if time allows :(

### CODE REVIEW

File: src/components/map/layers/CartoLayers.vue [hunk 1]
```diff
@@ -3,16 +3,26 @@
     <ol-source-osm ref="osmSource"/>
   </ol-tile-layer>
   <CustomWMTSLayer v-for="layer in mapStore.getLayers" :key="layer.name" :layer="layer"/>
-  <ol-tile-layer>
-    <ol-source-xyz ref="cavesSource" url="https://services7.arcgis.com/V2VriwTjJDabpGg6/ArcGIS/rest/services/2022_marec_export_ekataster_tile_layer/MapServer/WMTS/tile/1.0.0/2022_marec_export_ekataster_tile_layer/default/default028mm/{z}/{y}/{x}.png"/>
-  </ol-tile-layer>
+  <ol-vector-layer ref="cavesSource"  >
```
Comment (jeancaffou) [inline src/components/map/layers/CartoLayers.vue:6]: extra spaces

File: src/components/map/layers/CustomWMTSLayer.vue [hunk 2]
```diff
@@ -34,23 +36,37 @@ export default {
...
-        const image = tile.getImage()
+      options.tileLoadFunction = async (imageTile, src) => {
+        const image = imageTile.getImage()
         const storedTile = await db.tiles.where('tileKey')
-          .equals(url)
+          .equals(src)
           .first()
         if (!storedTile) {
-          image.src = url
+          const xhr = new XMLHttpRequest()
```
Comment (jeancaffou) [inline src/components/map/layers/CustomWMTSLayer.vue:45]: would probably be better if this was moved to a function, or use axios library?

**Review by jeancaffou — APPROVED** [review]
Not much to add, you're a Vue Pro 👍

**Review by jeancaffou — APPROVED** [review]

File: src/components/map/layers/CustomWMTSLayer.vue (hunk 2 above)
Comment (MaticDiba) [reply src/components/map/layers/CustomWMTSLayer.vue]: Good point. Created axios extension method to cover two cases where `XMLHttpRequest` was used

**Comment by MaticDiba** [comment]
> Not much to add, you're a Vue Pro 👍

Not sure about being Vue Pro. If nothing else, it's painful again to get back to a world without types :( it's a cold place ...

**Pull request merged**